docs/extensions.rst

```diff
@@ -77,3 +77,46 @@ and treat it as if the client had made a request.
 
 The ASGI server should set the pseudo ``:authority`` header value to
 be the same value as the request that triggered the push promise.
+
+Zero Copy Send
+------------------
+
+Zero copy send allows you to send the contents of a file descriptor to the
+HTTP client with zero copy. ASGI servers that implement this extension will
```

Comment:
Let's elaborate on "zero copy" - something like "zero copy (where the underlying OS directly handles the data transfer from a source file or socket without loading it into Python and out again)"

Reply:
good idea.